src/PrestaShopBundle/Controller/Admin/Sell/CustomerService/CustomerThreadController.php

```diff
@@ -66,13 +69,29 @@ class CustomerThreadController extends FrameworkBundleAdminController
      */
     public function indexAction(Request $request, CustomerThreadFilter $filters): Response
     {
+        $customerServicesSummaryQueryHandler = $this->get('prestashop.core.domain.customer_service.query_handler.GetCustomerServicesSummaryHandler');
+        $customerThreadKpiFactory = $this->get('prestashop.core.kpi_row.factory.customer_thread');
         $customerThreadGridFactory = $this->get('prestashop.core.grid.factory.customer_thread');
         $customerThreadGrid = $customerThreadGridFactory->getGrid($filters);
 
+        $params = [
+            $this->trans('Total threads', 'Admin.Catalog.Feature') => $all = CustomerThread::getTotalCustomerThreads(),
+            $this->trans('Threads pending', 'Admin.Catalog.Feature') => $pending = CustomerThread::getTotalCustomerThreads('status LIKE "%pending%"'),
+            $this->trans('Total number of customer messages', 'Admin.Catalog.Feature') => CustomerMessage::getTotalCustomerMessages('id_employee = 0'),
+            $this->trans('Total number of employee messages', 'Admin.Catalog.Feature') => CustomerMessage::getTotalCustomerMessages('id_employee != 0'),
+            $this->trans('Unread threads', 'Admin.Catalog.Feature') => $unread = CustomerThread::getTotalCustomerThreads('status = "open"'),
+            $this->trans('Closed threads', 'Admin.Catalog.Feature') => $all - ($unread + $pending),
+        ];
+
         return $this->render('@PrestaShop/Admin/Sell/CustomerService/CustomerThread/index.html.twig', [
             'help_link' => $this->generateSidebarLink($request->attributes->get('_legacy_controller')),
             'customerThreadGrid' => $this->presentGrid($customerThreadGrid),
             'enableSidebar' => true,
+            'categories' => Contact::getCategoriesContacts(),
```

Comment:
I don't understand why you need to get this list of category, since the `servicesSummary` query is already based on this list This result in weird things in your layout where you loop through the categories to get their ID and fetch the summary from the `servicesSummary` variable

You don't need all this complexity, the `servicesSummary` array returned should be constructed so that you have all the necessary elements, including the category name/description if you need it